### PR TITLE
Setup sections for copy

### PR DESCRIPTION
**Pull request by frm**

### CODE REVIEW

**Review by jfranciscosousa — CHANGES_REQUESTED** [review]
Remove `yarn.lock` we run `npm` in this project. Also don't forget to lint with `npm run lint-fix` and `npm run lint-styles-fix` if needed.

**Comment by jfranciscosousa** [comment]
@fribmendes want to change spacings after or in this PR?

**Comment by frm** [comment]
After this PR

**Comment by jfranciscosousa** [comment]
@luisptorres will style this when he finds some time for it.

File: src/sections/Contact/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Heading from "../../components/Heading";
+import Text from "../../components/Text";
+
+export default function Contact() {
```
Comment (jfranciscosousa) [inline src/sections/Contact/index.js]: Useless section. Add the missing `mail` contact to the `SocialLinks` component,

**Review by jfranciscosousa — CHANGES_REQUESTED** [review]

File: src/sections/Contact/index.js (hunk 1 above)
Comment (jfranciscosousa) [reply src/sections/Contact/index.js]: `Footer` component, my mistake.

**Comment by nelsonmestevao** [comment]
@fribmendes, my name is N**e**lson Estevão :stuck_out_tongue_winking_eye:

**Comment by frm** [comment]
@luisptorres I'd rather not keep this on hold waiting for you. I suggest merging right away and you can add style on a different PR

**Comment by frm** [comment]
@jfranciscosousa @nelsonmestevao all done 👍

**Review by jfranciscosousa — APPROVED** [review]

**Pull request merged**